src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -53,7 +53,7 @@ public static void dropBqQuery(String table) throws IOException, InterruptedExce
     executeQuery(dropQuery);
   }
 
-  private static int executeQuery(String query) throws InterruptedException, IOException {
+  public static int executeQuery(String query) throws InterruptedException, IOException {
```

Comment:
Why this change has been introduced?

Reply:
To execute query other than listed in CDAP framework, we need to call this method. I need to execute where query in ODP but for that we need to make it public.